extensions/Cake.ReportPortal.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.ReportPortal
+NuGet: Cake.ReportPortal
```

Comment:
Looks like this package has been superseded by the package `Cake.Buns.ReportPortal` from the same author.
https://github.com/OlegYanushkevich/Cake.Buns.ReportPortal
https://www.nuget.org/packages/Cake.Buns.ReportPortal/

Let's blacklist this one

Reply:
Adding this addin to exclusion list: : https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194